### PR TITLE
Linux Installation Guide

### PR DESCRIPTION
**Pull request by nethad**
A linux installation guide (done with Ubuntu 24.04 LTS, which is the recommended distro).

Open question for me as a first-time contributor:

- [ ] Is it OK to put this under "Tutorials"?
- [ ] Is the configuration otherwise correct?
- [ ] How can we add this to the 5.1 release instead of next?
- [ ] Should it be translated to English as well?

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -11,26 +11,26 @@
 
 module.exports = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
+  tutorialSidebar: [{ type: "autogenerated", dirName: "." }],
```
Comment (nethad) [inline sidebars.js:14]: My editor has an autoconfiguration here, should I revert this or is it OK?

**Comment by nucleus-ffm** [comment]
I invited you to the manual team to give you access to the repository. I would also merge the pull request to the branch linuxInstallationGuide first, as this is more convenient to work on and then create a new Pull-Request.

 > Is it OK to put this under "Tutorials"?

I would suggest creating a new category "Installation".

> Is the configuration otherwise correct?

Right now it looks good, but I still have to build it locally to test it

> How can we add this to the 5.1 release instead of next?

The version 5.1.0 docs can be found under https://github.com/Ultraschall/ultraschall-manual/tree/main/versioned_docs/version-5.1.0

> Should it be translated to English as well?

Yes, please. The most of the docs are translated with Deepl. If you want, you can use this too.

**Review by nucleus-ffm — APPROVED** [review]
Looks good.

**Comment by nethad** [comment]
@nucleus-ffm OK cool, thanks for the answer, I'll work on the new PR then :+1: 

https://github.com/Ultraschall/ultraschall-manual/pull/50

**Pull request merged**